src/helpers/dimensions.ts

```diff
@@ -303,6 +303,18 @@ function getScatterDimensions(mdObject: VisualizationObject.IVisualizationObject
     ];
 }
 
+export function getGeoChartDimensions(buckets: VisualizationObject.IBucket[]): AFM.IDimension[] {
+    const { attributes = [] }: AFM.IAfm = convertBucketsToAFM(buckets);
+    return [
+        {
+            itemIdentifiers: [MEASUREGROUP],
+        },
+        {
+            itemIdentifiers: attributes.map((attr: AFM.IAttribute) => attr.localIdentifier),
+        },
+    ];
+}
+
```

Comment:
How about incase _Geo Pushpin has 2 attributes (A from location, B from segment), 1 measure (M1 from measure size)_
=> result should be
```
[
  { itemIdentifiers: [ 'A' ] },
  { itemIdentifiers: [ 'B', 'measureGroup' ] }
]
```